Cargo.toml

```diff
@@ -42,5 +43,9 @@ box_drawing = "0.1.2"
 textwrap = { default-features = false, version = "0.15.0" }
 cargo_toml = "0.11.5"
 
+# Why does cargo update this dependency to the latest version
+# `clap_lex` uses 6.0 (https://github.com/clap-rs/clap/blob/master/clap_lex/Cargo.toml#L41)
+os_str_bytes = { default-features = false, version = "=6.0", features = ["raw_os_str"] }
```

Comment:
Do we need to do this? Is there no workaround we can do?

Reply:
There is a workaround. By ignoring the Cargo lockfile (`Cargo.lock`) from the repository. 

- https://github.com/rust-lang/cargo/issues/315
- https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html#:~:text=If%20you%E2%80%99re%20building%20a%20non%2Dend%20product%2C%20such%20as%20a%20rust%20library%20that%20other%20rust%20packages%20will%20depend%20on%2C%20put%20Cargo.lock%20in%20your%20.gitignore.

Reply:
This project shouldn't ignore Cargo.lock. Does this dependency matter so much that it would increase MSRV?

Reply:
It would increase it to `1.57.0`.

Reply:
That's fine IMHO.